azavea/climate-change-lab
# Feature/kak/get api call button#181

## Overview

Adds a popup for user to copy curl command to get chart data.

### Demo

![image](https://user-images.githubusercontent.com/960264/28926459-dad5fc52-7835-11e7-8bae-80794d5fefa8.png)


### Notes

Needs a fontello icon for clipboard copy. Existing project icons don't seem to have anything appropriate.


## Testing Instructions

 * Click to copy curl command below chart
 * Should get confirmation tooltip that it copied
 * Should be able to run curl command and get data

Closes #181
